_config.ts

```diff
@@ -210,13 +176,35 @@ site.ignore(
   // "subhosting",
 );
 
-site.scopedUpdates((path) => path == "/overrides.css");
-site.use(
-  checkUrls({
-    external: false, // Set to true to check external links
-    output: "_broken_links.json",
-    ignore: ["https://www.googletagmanager.com"],
-  }),
+site.scopedUpdates(
+  (path) => path == "/overrides.css",
+  (path) => /\.(js|ts)$/.test(path),
+  (path) => path.startsWith("/api/deno/"),
 );
 
+const SKIP_CHECK_URLS = (Deno.env.get("SKIP_CHECK_URLS") || "false")
+  .toLowerCase();
+
+if (SKIP_CHECK_URLS !== "true") {
```

Comment:
good point, will add env variable

Reply:
Wait, if there is no setting it will default to "don't skip", so it will still run if you use deno task serve, and it will run on CI
My thought being that on deploy the extra 5 seconds that the link checker takes probably are fine... (maybe)